code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -144,6 +151,8 @@
 		temperature = reference.temperature
 
 	// Handles the burning of atoms.
+	if(temperature < FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
```

Comment:
Rather then sticking checks like this and `if(temperature < FREON_MAXIMUM_BURN_TEMPERATURE)` all over the place, it'd be better if you had a var on the hotspot called like "cold_burn" or whatever, and then set that on process. 
Would make the semi messy checks a bit esier to read